source/Nevermore/Advanced/Concurrency/NoOpConcurrencyHandler.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Nito.Disposables;
+
+namespace Nevermore.Advanced.Concurrency
+{
+    public class NoOpConcurrencyHandler : ITransactionConcurrencyHandler
+    {
+        public IDisposable Lock()
+        {
+            return NoopDisposable.Instance;
+        }
+
+        public async Task<IDisposable> LockAsync(CancellationToken cancellationToken)
+        {
+            await Task.Yield();
```

Comment:
AFAIK Task.Yield incurs a thread-jump. Given we aren't trying to use it to capture Task.Current anymore, looks like we could replace with `return Task.FromResult(NoopDisposable.Instance)`. Nevermore doesn't have the weird "must async everything" convention that Server does.